source/globalCommands.py

```diff
@@ -4931,6 +4933,72 @@ def _enableScreenCurtain(doEnable: bool = True):
 					return
 				_enableScreenCurtain()
 
+	@script(
+		description=_(
+			# Translators: Describes a command.
+			"Toggles the magnifier on and off."
```

Comment:
please make sure all descriptions in this file start with a capital and don't end with a period

```suggestion
			"Toggles the magnifier on and off"
```